app/parse.py

```diff
@@ -15,8 +30,102 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+class WebScraper:
+    def __init__(self) -> None:
+        self.options = Options()
+        self.options.add_argument("--headless")
+        self.driver = webdriver.Chrome(options=self.options)
+
+    def click_accept_cookies(self, url: str) -> None:
+        self.driver.get(url)
+        cookies_accept_button = self.driver.find_element(
+            By.CLASS_NAME, "acceptCookies"
+        )
+        if cookies_accept_button.is_displayed():
+            cookies_accept_button.click()
+
+    def scroll_down_page_with_more_button(self, url: str) -> None:
+        self.driver.get(url)
+
+        while True:
+            try:
+                more_button = self.driver.find_element(
+                    By.CSS_SELECTOR, "[class*=scroll-more]"
+                )
+                if more_button.is_displayed():
+                    more_button.click()
+                    time.sleep(1)
```

Comment:
is there a possibility that error will be raised in try block?
If no, then it's better to use if some statement